### PR TITLE
Enhance score http client

### PR DESCRIPTION
**Pull request by mihaitusa**
Closes https://github.com/CloudSlang/score-actions/issues/109

### CODE REVIEW

**Comment by mihaitusa** [comment]
@Bonczidai Please review it!

File: content/io/cloudslang/base/network/rest/http_client_action.sl [hunk 1]
```diff
@@ -320,6 +320,7 @@ operation:
     - error_message: ${returnResult if returnCode != '0' else ''}
     - return_code: ${returnCode}
     - status_code: ${'' if 'statusCode' not in locals() else statusCode}
+    - response_headers: ${'' if 'responseHeaders' not in locals() else responseHeaders}
```
Comment (Bonczidai) [inline content/io/cloudslang/base/network/rest/http_client_action.sl]: get('responseHeaders', '')

File: content/io/cloudslang/base/network/utils/get_header_value.sl [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        begin_index = response_headers.find(header_name + ':')
+        if begin_index != -1:
+          response_headers = response_headers[begin_index + len(header_name) + 2:]
+          result = response_headers.split(' ')[0]
+        else:
+          error_message = 'Could not find specified header: ' + header_name
+      except Exception as exception:
+        error_message = exception
+  outputs:
+    - result
```
Comment (Bonczidai) [inline content/io/cloudslang/base/network/utils/get_header_value.sl]: maybe handle `result` initialization before `if`

**Comment by mihaitusa** [comment]
@Bonczidai  Please take a short look again so we can merge it! Thanks.

**Comment by Bonczidai** [comment]
test

**Comment by mihaitusa** [comment]
@Bonczidai  WDYT... should we merge it?

**Pull request merged**